### DISCUSSION TITLE
Welcome to SigNoz discussions

### DISCUSSION THREAD

**Discussion by pranay01** [SigNoz/signoz#14] [General]
Hey folks 👋

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by praveeshvp** [comment]
Hello Team,

I'm a Software Architect from a Cruise line company and we are looking for an Open source APM tool for our onboard ship. I do have some questions to ask, hence joining here.

Regards,
Praveesh

**Reply by makeavish** [reply]
Great you can shoot your questions here or in [Slack community](https://signoz.io/slack).
